smoke-tests/designer/wdio.headless.conf.js

```diff
@@ -1,6 +1,7 @@
 const { config } = require("./wdio.conf.js");
 
 (config.baseUrl = "http://localhost:3000"), // TODO:- Change to the url in the CI environment
```

Comment:
this should be fine btw. The services are starting at 0.0.0.0:3000/3009. Because we're not configuring ports as an environment variable, and we aren't running your smoke tests within the container, this will be the best you get..!